From 59e15ab918ee1823e1d289585d75686032a08909 Mon Sep 17 00:00:00 2001
From: Jeremy Newton <Jeremy.Newton@amd.com>
Date: Mon, 25 Apr 2022 12:33:46 -0400
Subject: [PATCH 4/4] SWDEV-336248 - Don't exclude cl_egl.h from install

cl_icd.h includes cl_egl.h, but cl_egl.h is excluded from install. It
was reported by the community that including cl_icd.h would error due to
the missing cl_egl.h file.

I'm assuming that cl_egl.h was excluded because EGL isn't important for
ROCm. Since cl_icd.h has some important typedefs, it makes sense just
to include cl_egl.h, which seems to be a common practice in open source
projects.

Signed-off-by: Jeremy Newton <Jeremy.Newton@amd.com>
Change-Id: I506257c2dc51512ec8e11b1e5dadbe6e48ad785b
---
 packaging/CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packaging/CMakeLists.txt b/packaging/CMakeLists.txt
index b6d206a..60050f3 100644
--- a/packaging/CMakeLists.txt
+++ b/packaging/CMakeLists.txt
@@ -18,8 +18,7 @@ install(DIRECTORY ${CMAKE_SOURCE_DIR}/khronos/headers/opencl2.2/CL
         USE_SOURCE_PERMISSIONS
         PATTERN cl_d3d10.h EXCLUDE
         PATTERN cl_d3d11.h EXCLUDE
-        PATTERN cl_dx9_media_sharing.h EXCLUDE
-        PATTERN cl_egl.h EXCLUDE )
+        PATTERN cl_dx9_media_sharing.h EXCLUDE )
 
 if(BUILD_ICD)
   install(TARGETS OpenCL DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT icd )
-- 
2.34.1

